packages/react-components/lib/robots/robot-info.tsx

```diff
@@ -15,6 +15,16 @@ function getTaskStatusDisplay(assignedTask?: string, taskStatus?: string) {
   }
 }
 
+function calculateLastUpdateTime(updateTime: number): string {
+  if (lastUpdateTime === updateTime) {
+    return ((Date.now() - timeOfLastUpdate) / 1000).toFixed(1);
+  } else {
+    lastUpdateTime = updateTime;
+    timeOfLastUpdate = Date.now();
+    return '< 1';
+  }
+}
+
```

Comment:
This should use data-fns to provide better relative time formatting.

Reply:
oh! `data-fns` looks amazing! That would be awesome